modules/administration_manual/pages/installation/manual_installation_ubuntu_16_04.adoc

```diff
@@ -0,0 +1,163 @@
+= Manual Installation on Linux - Ubuntu 16.04
+:nginx-app-tracing-url: https://www.nginx.com/blog/application-tracing-nginx-plus/
+
+== Introduction
+This guide describes how to install the required dependencies, the database and ownCloud itself in a
+single server setup.
+
+
+[[install-the-required-packages]]
+== Prepare your Server
+
+*Necessary steps to prepare the installation of ownCloud*
+
+Before we can start to install ownCloud, we need to make sure that everything needed
+is available.
+
+Steps to take:
+
+* Install MariaDB Database
+* Install Redis Server
+* Install Apache and the necessary dependencies
+* Configure Apache
+* Download and install ownCloud 
+
+
+=== Install MariaDB Database
+
+Perform the following steps to install MariaDB and setting up a Database for ownCloud.
+
+....
+apt update
+apt install -y mariadb-server-10.0
+....
+After this has finish, create the database and a database user for ownCloud to use:
+
+....
+mysql -u root
+
+MariaDB [(none)]> create database owncloud;
+MariaDB [(none)]> GRANT USAGE ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> GRANT ALL PRIVILEGES ON owncloud.* TO owncloud@localhost IDENTIFIED BY 'password';
+MariaDB [(none)]> flush privileges;
+MariaDB [(none)]> exit
+....
+
+=== Install Redis Server
+Redis in needed for Transactional File Locking in ownCloud. Perform the following step to install it:
+....
+apt install -y redis-server
+....
+
+=== Install Apache and the necessary dependencies
+The following steps will install the Apache Webserver and the dependencies needed for ownCloud:
+....
+apt install -y \
+apache2 \
+libapache2-mod-php \
+php-gd \
+php-json \
+php-mysql \
+php-curl \
+php-intl \
+php-mcrypt \
+php-imagick \
+php-zip \
+php-xml \
+php-mbstring \
+php-soap \
+php-ldap \
+php-apcu \
+php-redis \
+smbclient \
+php-smbclient \
+gettext-base && \
+a2enmod ssl rewrite headers env dir mime ldap && \
+service apache2 restart
+....
+
+=== Configure Apache
+We will create a new apache config file for ownCloud:
+....
+vim /etc/apache2/sites-available/owncloud.conf
+....
+Please add the following to the file and adjust parameters like SERVER-NAME and
+the paths to the certificates.
```

Comment:
what if the user does not have certificates? I suggest we have 2 files, one for http only and one for https

Reply:
It'd be good to strongly encourage users to use HTTPS, but, not all users may see the value in that option nor want to put in the effort to implement it. So, I agree that both an HTTP and HTTPS option should be made available.